compiler-core/src/javascript/tests/snapshots/gleam_core__javascript__tests__bit_arrays__match_literal_unaligned_float.snap

```diff
@@ -0,0 +1,33 @@
+---
+source: compiler-core/src/javascript/tests/bit_arrays.rs
+expression: "\nfn go(x) {\n  let n = 1\n  let assert <<_:size(n), 1.1, _:bits>> = x\n}\n"
+---
+----- SOURCE CODE
+
+fn go(x) {
+  let n = 1
+  let assert <<_:size(n), 1.1, _:bits>> = x
+}
+
+
+----- COMPILED JAVASCRIPT
+import { makeError, bitArraySliceToFloat } from "../gleam.mjs";
+
+function go(x) {
+  let n = 1;
+  if (
+    x.bitSize < n ||
+    x.bitSize < 64 + n ||
```

Comment:
Previously Gears made it not output these redundant checks, it would be fab to retain that!